File: 217-contains-duplicate/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        std::set <int> check;
        for(int i = 0; i < nums.size(); i++){
            check.insert(nums[i]);
        }
        if (check.size() != nums.size()){
            return true;
        }
        return false;
    }
};

File: 217-contains-duplicate/contains-duplicate.rs
use std::collections::HashSet;
impl Solution {
    pub fn contains_duplicate(nums: Vec<i32>) -> bool {
        let mut check = HashSet::new();
        for &num in &nums{
            if !check.insert(num){
                return true;
            }
        }

        false
    }
}